From b2d26322aa9e2b958833f652ee9f7c97cdf28552 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 3 Aug 2015 22:00:17 +0100
Subject: [PATCH 4/4] Output line numbers in callstacks

Now just a marker for where to add the new code.
---
 Source/cmMakefile.cxx | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Source/cmMakefile.cxx b/Source/cmMakefile.cxx
index 78cae0e986..8041b0a690 100644
--- a/Source/cmMakefile.cxx
+++ b/Source/cmMakefile.cxx
@@ -4037,6 +4037,9 @@ std::string cmMakefile::FormatListFileStack() const
   cmStateSnapshot snp = this->StateSnapshot;
   while (snp.IsValid()) {
     listFiles.push_back(snp.GetExecutionListFile());
+//  GetEntryPointLine() has been removed, replaced with something a lot more complicated,
+//  this patch is reduced to a marker for where the replacement code should be added.
+//  listFiles.push_back(snp.GetExecutionListFile() + ":" + static_cast<std::ostringstream*>( &(std::ostringstream() << snp.GetEntryPointLine()) )->str());
     snp = snp.GetCallStackParent();
   }
   std::reverse(listFiles.begin(), listFiles.end());
-- 
2.31.0.windows.1

